client/src/pages/Notifications/Notifications.tsx

```diff
@@ -59,9 +64,10 @@ export default function Notifications(): JSX.Element {
   };
 
   return loggedInUser ? (
-    <div>
+     <>
       <CssBaseline />
-      <AuthHeader linkTo={`/create-contest`} btnText="CREATE CONTEST" />
+      {header ? <AuthHeader linkTo={`/create-contest`} btnText="CREATE CONTEST" /> : ''}
+      <Box className={classes.root}>
```

Comment:
The Box component has flexbox built into it, you can add the props here rather than using a class to style it :)

Reply:
addressed with the recent commit :)